streaming/plugins/abci/v1/examples/plugin-python/kafka.py

```diff
@@ -0,0 +1,97 @@
+from concurrent import futures
```

Comment:
Do we really need to store the python example in the repo?
We don't have any CI to test that it actually works. I feel like it could be put somewhere else.

Reply:
No they don't need to be in there. I'll remove the python code.